src/Symfony/Component/DependencyInjection/Compiler/AnalyzeServiceReferencesPass.php

```diff
@@ -155,6 +156,10 @@ private function getDefinitionId($id)
     private function getExpressionLanguage()
     {
         if (null === $this->expressionLanguage) {
+            if (!class_exists('Symfony\Component\ExpressionLanguage\ExpressionLanguage')) {
```

Comment:
why not : 
```
if (!class_exists(ExpressionLanguage::class)) {
//body
}
```

Reply:
copy/pasting from the other check, that's it :)

Reply:
Yeah but I think it's better to use this since we already replaced some of the one in the symfony code base.